skll/utilities/run_experiment.py

```diff
@@ -28,66 +28,76 @@ def main(argv=None):
     :type argv: list of str
     """
     # Get command line arguments
-    parser = argparse.ArgumentParser(
-        description="Runs the scikit-learn experiments in a given config file.\
-                     If Grid Map is installed, jobs will automatically be \
-                     created and run on a DRMAA-compatible cluster.",
-        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
-        conflict_handler='resolve')
+    parser = argparse.ArgumentParser(description='Runs the scikit-learn '
```

Comment:
Nitpick: To save on space, it might be slightly better to just import `ArgumentParser`/`ArgumentDefaultsHelpFormatter` directly.